### PR TITLE
2024 Program

### LINKED ISSUE #61: Add program page (with presenters below)

### PR DESCRIPTION
**Pull request by garrying**
Closes https://github.com/ournetworks/2024/issues/61

### CODE REVIEW

File: _data/sessions.yml [hunk 1]
```diff
@@ -1585,3 +1585,36 @@
...
+      - brooklyn
+    sessionType: talk
+    description: >
+      Session description text
+
+  - sessionID: 24004
+    title: 'Session title 4'
+    presenterID:
+      - michelle
+    sessionType: panel
```
Comment (dcwalk) [inline _data/sessions.yml]:
```suggestion
    sessionType: discussion
```

File: program.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        <td class="font-mono text time-cell">11:30 ↓<br />13:00 </td>
+        <td colspan="3"><strong><span class="font-mono">LUNCH</span></strong></td>
+    </tr>
+    <tr>
+        <td class="font-mono text time-cell">13:00 ↓<br />13:30</td>
+        <td colspan="3"><strong><span class="font-mono">PRESENT</span> / <a href="#brooklyn">Brooklyn Zelenka</a> </strong></td>
+    </tr>
+    <tr>
+        <td class="font-mono text time-cell" rowspan="2">13:40 ↓<br />15:00</td>
+        <td colspan="3"><strong><span class="font-mono">PRESENT</span> / <a href="#joni">Joni Schinkel</a> </strong></td>
```
Comment (dcwalk) [inline program.md]: shouldn't these not be hardcoded?

Comment (dcwalk) [reply program.md]: also this session should go in one of the rooms as a "workshop"

File: program.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        <td><em>Reach</em><br />Open</td>
+        <td><em>Spillover</em><br />Open</td>
+    </tr>
+    <tr>
+        <td class="font-mono text time-cell">15:00 ↓<br />15:30 </td>
+        <td colspan="3"><strong>BREAK</strong></td>
+    </tr>
+    <tr>
+        <td class="font-mono text time-cell">15:30 ↓<br />16:00</td>
+        <td colspan="3"><strong><span class="font-mono">FUTURE</span> / <a href="#michelle">Michelle Kaczmarek</a></strong></td>
```
Comment (dcwalk) [inline program.md]: likewise this should be in one of the rooms, and we are waiting on speaker details for EJ here

File: program.md (hunk 2 above)
Comment (dcwalk) [inline program.md]: but also wondering if we should move joni earlier to morning? I think then there is more time?

Comment (garrying) [reply program.md]: > shouldn't these not be hardcoded?

Sticking with hardcoded for now—I find it a bit easier to work with at this stage.

File: program.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+        <td colspan="3"><strong><span class="font-mono">PRESENT</span> / <a href="#brooklyn">Brooklyn Zelenka</a> </strong></td>
+    </tr>
+    <tr>
+        <td class="font-mono text time-cell">13:40 ↓<br />15:00</td>
+        <td colspan="3">
+            <strong>Lightning Talks</strong>
+            <table class="w-100">
+                <tr class="border-0">
+                    <td><em class="font-display-mono uppercase text-sm whitespace-nowrap not-italic tracking-wide">⌂ Sound</em><br />
+                        <ul class="mt-0 mb-0 pl-1">
```
Comment (dcwalk) [inline program.md]:
```suggestion

```

Comment (dcwalk) [reply program.md]: remove these for now

File: program.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+                </tr>
+            </table>
+        </td>
+    </tr>
+    <tr>
+        <td class="font-mono text time-cell">11:30 ↓<br />13:00 </td>
+        <td colspan="3"><strong><span class="font-mono">LUNCH</span></strong></td>
+    </tr>
+    <tr>
+        <td class="font-mono text time-cell">13:00 ↓<br />13:30</td>
```
Comment (dcwalk) [inline program.md]:
```suggestion
        <td class="font-mono text time-cell">13:00 ↓<br />13:40</td>
```

File: program.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+        <td colspan="3"><strong><span class="font-mono">PRESENT</span> / <a href="#brooklyn">Brooklyn Zelenka</a> </strong></td>
+    </tr>
+    <tr>
+        <td class="font-mono text time-cell">13:40 ↓<br />15:00</td>
```
Comment (dcwalk) [inline program.md]:
```suggestion
        <td class="font-mono text time-cell">13:50 ↓<br />15:00</td>
```

File: program.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+                    <td><em class="font-display-mono uppercase text-sm whitespace-nowrap not-italic tracking-wide">⌂ Sound</em><br />Roundtable - Repair Futures <a href="#michelle">Michelle Kaczmarek</a></td>
+                    <td><em class="font-display-mono uppercase text-sm whitespace-nowrap not-italic tracking-wide">⌂ Reach</em><br />Workshop - Small File <a href="#joni">Joni Schinkel</a></td>
+                    <td><em class="font-display-mono uppercase text-sm whitespace-nowrap not-italic tracking-wide">⌂ Spillover</em><br />Open</td>
+                </tr>
+            </table>
+        </td>
+    </tr>
+    <tr>
+        <td class="font-mono text time-cell">17:30 ↓<br />18:00</td>
+        <td colspan="3">Closing Keynote</td>
```
Comment (dcwalk) [inline program.md]:
```suggestion
        <td colspan="3">Closing</td>
```

**Pull request merged**